jsk_tools/env-hooks/99.jsk_tools.sh

```diff
@@ -165,3 +165,27 @@ rost() {
         fi
     fi
 }
+
+restart_travis() {
+  # Restart travis from command line
+  if [ $# -lt 2 ]; then
+    echo "usage: restart_travis <repo_slug> <job_id>"
+    echo "example:"
+    echo "  restart_travis jsk-ros-pkg/jsk_common 1258.2"
+    return 1
+  fi
+  if [ -z $SLACK_TOKEN ]; then
+    echo "Please set SLACK_TOKEN (see: https://api.slack.com/web)"
+    return 1
+  fi
+  if [ -z $SLACK_USERNAME ]; then
```

Comment:
<img width="646" alt="screen shot 2015-09-08 at 1 08 43 am" src="https://cloud.githubusercontent.com/assets/4310419/9720132/30587fcc-55c6-11e5-905a-76ee38d7461e.png">

The restart does not responds to the msg of bot. I think this is because bot is not the member of travis channel.
![image](https://cloud.githubusercontent.com/assets/4310419/9720152/6c277558-55c6-11e5-982a-9a7d61a4d664.png)